### ISSUE TITLE
ThreeDSecureV1UiCustomization doesn't support accented characters

### ISSUE THREAD

**Issue by calvarez-ov** [braintree/braintree_android#288]
### General information

* SDK/Library version: 3.8.0
* Environment: Sandbox
* Android Version and Device: Android 9, Galaxy S9
* Braintree dependencies: 
  - com.braintreepayments.api:braintree:3.8.0
  - com.braintreepayments.api:three-d-secure:3.8.0
  - com.braintreepayments.api:drop-in:4.5.0

### Issue description
```kotlin
val threeDSecuireUiCustomization = ThreeDSecureV1UiCustomization().apply {
    redirectButtonText("CLICK HERE")
    redirectDescription("Some téxt with accéntéd chàractèrs")
}
val request = ThreeDSecureRequest().apply {
    versionRequested(ThreeDSecureRequest.VERSION_2)
    nonce(nonce)
    amount(amount)
    email("me@me.com")
    mobilePhoneNumber("+33123456789")
    billingAddress(postalAddress)
    additionalInformation(ThreeDSecureAdditionalInformation().shippingAddress(postalAddress))
    v1UiCustomization(threeDSecuireUiCustomization)
}
ThreeDSecure.performVerification(fragment, request) {...}
```    

Expected behavior: After the user enters the 3ds code, the user should see the page with the big blue button with label "CLICK HERE" and explanation text "Some téxt with accéntéd chàractèrs"

Actual behavior:
404 not found error

<img src="https://user-images.githubusercontent.com/9365138/73171816-ede1ea80-4101-11ea-84fc-12e1a4549a66.jpg" width="320">

Note: the expected behavior is seen if we don't include any accented characters in the button label or description. The buggy behavior is seen if either the button label or the description contain accented characters.

**Comment by sestevens** [comment]
@calvarez-ov Thanks for reporting this. We're able to reproduce the issue, and we're currently working to resolve it. We'll update here once we have a fix.

**Comment by sestevens** [comment]
@calvarez-ov This bug should be resolved in version 3.9.0.
